LICENSE.txt

```diff
@@ -1,91 +1,270 @@
-==============================================================================
-compiler-builtins License
-==============================================================================
+compiler-builtins as a whole is available for use under both the MIT license
+and the Apache-2.0 license with the LLVM exception (Apache-2.0 WITH
+LLVM-exception AND MIT).
```

Comment:
```suggestion
compiler-builtins as a whole is available for use under both the MIT license
and the Apache-2.0 license with the LLVM exception (`MIT AND Apache-2.0 WITH
LLVM-exception`).
```

Minor consistency and readability nit. Not a blocker.

Reply:
Updated